src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -348,6 +388,37 @@ private void EnsureSignature()
             }
         }
 
+        private void EnsureAccessorsBinding()
```

Comment:
>EnsureAccessorsBinding [](http://example.com/codeflow?start=21&length=22)

I think it would be better to use a name that reflects the purpose of this method, rather than how the purpose is achieved. It looks like the purpose is to ensure that a backing field is synthesized if it is supposed to be synthesized. The same comment applies to the names of added CompletionPart members.